projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.html

```diff
@@ -3,89 +3,179 @@ <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
 </h2>
 <ng-container *ngIf="quotes$ | async as quotes">
   <ng-container *ngIf="quotesLoaded$ | async; else loading">
-    <div class="row" *ngIf="quotes.length > 0; else noQuotes">
-      <div *ngFor="let quote of quotes" class="col-12 col-md-6 d-flex">
-        <div class="info-card">
-          <h6 class="title background-color-4">
-            {{ quote.defaultCategory?.name }}
-          </h6>
-          <div class="info-card-wrapper">
-            <div class="info-card-content">
-              <div class="info-card-details">
-                <div class="info-card-data">
-                  <span class="label">{{ quote.planName }}</span
-                  >:
-                  <span class="value">{{ quote.quoteId }}</span>
-                </div>
-                <ng-container *ngIf="quote.quoteWorkflowStatus">
+    <div *ngIf="quotes.length > 0; else noQuotes">
+      <div class="row justify-content-between align-items-center mb-5">
+        <div class="col-md-5 d-flex">
+          <label class="col-8 pr-3 px-0">
+            <span class="label-content">{{
+              'quote.productCategory' | cxTranslate
+            }}</span>
+            <ng-select
+              placeholder="{{ 'quote.selectCategory' | cxTranslate }}"
+              [searchable]="false"
+              [clearable]="false"
+              [items]="options"
+              bindLabel="name"
+              bindValue="code"
+              (change)="selectCategory($event)"
+            >
+            </ng-select>
+          </label>
+          <button
+            class="btn btn-link cx-action-link notice align-self-end pb-4"
+            (click)="quoteCodesForCompare = []; selectedQuote = null"
+            [disabled]="quoteCodesForCompare?.length === 0"
+          >
+            {{ 'quote.clearAll' | cxTranslate }}
+          </button>
+        </div>
+        <div class="col-xs-12 col-sm-3 mt-4">
+          <button
+            class="col-12 primary-button btn-block"
```

Comment:
`primary-button btn-block`. Button should almost never contain `col` classes as they add extra padding

Reply:
Already pushed this change.